code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -439,3 +439,53 @@
 
 /obj/item/reagent_containers/glass/bottle/morphine/sleeper
 	cap_on = FALSE
+
+//types of syrups
+
+/obj/item/reagent_containers/food/drinks/bottle/syrup_bottle/caramel
+	name = "bottle of caramel syrup"
+	desc = "A pump bottle containing caramalized sugar, also known as caramel. Do not lick."
+	list_reagents = list(/datum/reagent/consumable/caramel = 50)
+
+/obj/item/reagent_containers/food/drinks/bottle/syrup_bottle/liqueur
+	name = "bottle of coffee liqueur syrup"
+	desc = "A pump bottle containing mexican coffee-flavoured liqueur syrup. In production since 1936, HONK."
+	list_reagents = list(/datum/reagent/consumable/ethanol/kahlua = 50)
+
+/obj/item/reagent_containers/food/drinks/bottle/syrup_bottle/korta_nectar
```

Comment:
Don't have this

Reply:
Yes I know I added it, unless you would like me to remove it?

Reply:
We don't have korta nuts so it doesn't make much sense.

Reply:
Fair enough. Removed.

Reply:
My food port should add this anyway.. will check later